src/lightbox/shareImage.js

```diff
@@ -33,10 +34,10 @@ export default async (url: string, auth: Auth) => {
       try {
         await Share.share({ url: uri, message: url });
       } catch (error) {
-        showToast('Can not share');
+        showToast('Share failed');
       }
     } catch (error) {
-      showToast('Can not download');
+      showToast('Share failed');
     }
```

Comment:
Making this about "sharing" in both cases makes sense.

This also means we can drop the inner try/catch -- it's now redundant with the outer one.